public/app/types/events.ts

```diff
@@ -28,6 +28,12 @@ export interface ShowModalReactPayload {
   props?: any;
 }
 
+export interface OpenExtensionSidebarPayload {
+  props?: any;
```

Comment:
Can we avoid using `any` here please?

Reply:
Changed to `Record<string, unknown>` in 887f7527c14f192b1c8afb41f10af9c65c1016aa

Reply:
Oh, sorry - e858fc23dc36c49da3eeb9704a6ace5138489fee to fix.